src/App.js

```diff
@@ -1,12 +1,25 @@
 import React from "react";
-
+import Registration from "./Components/Registration/registrationForm";
+import Dashboard from "../src/Components/Dashboard/Dashboard";
+import CreatePoll from "./Components/CreatePoll/CreatePoll";
+import Login from "./Components/Login/Login";
+import NavBar from "./navbar";
+import { BrowserRouter, Route } from "react-router-dom";
 import "./App.css";
 
 function App() {
   return (
-    <div className="App">
-      <h3>Lets Start Polling App</h3>
-    </div>
+    <React.Fragment>
+      <BrowserRouter>
+        <NavBar />
+
+        <Route exact path="/" component={Registration} />
+
+        <Route path="/login" component={Login} />
+        <Route path="/dashboard" component={Dashboard} />
```

Comment:
dashboard page will only available after login

Reply:
login page is not completed yet. I am working on Admin action first,as you suggested
will format the routing properly when login page completed